engines/docker/config.go

```diff
@@ -0,0 +1,26 @@
+package dockerengine
+
+import (
+	schematypes "github.com/taskcluster/go-schematypes"
+)
+
+type configType struct {
+	DockerSocket   string `json:"dockerSocket"`
+	MaxConcurrency int    `json:"maxConcurrency"`
+}
+
+var configSchema = schematypes.Object{
+	Properties: schematypes.Properties{
+		"dockerSocket": schematypes.String{
+			Title: "Docker Endpoint",
+			Description: "dockerEndpoint is the endpoint to use for communicating\n" +
```

Comment:
Try this:
```go
  Description: util.Markdown(`
    Path to the docker socket hosting the docker remote API.

    If not given default value 'unix:///var/run/docker.sock' will be used.
  `),
```

The `util.Markdown` function removes the indentation... so the strings looks nice. And it changes `'` into back-ticks that will be formatted with `<code>` in markdown. I know it wasn't smart to use single quotes for back-ticks, but I'll find a better solution in the future... We write markdown in JSON schema descriptions, in a not so distant future we'll render this on the docs site :)

This should encourage useful descriptions, so people can do more on their own...